collect_app/src/main/java/org/odk/collect/android/widgets/RangeDecimalWidget.java

```diff
@@ -40,45 +112,62 @@ public IAnswerData getAnswer() {
     }
 
     @Override
-    protected void setUpActualValueLabel() {
+    public void setOnLongClickListener(OnLongClickListener l) {
+    }
+
+    @Override
+    public boolean suppressFlingGesture(MotionEvent e1, MotionEvent e2, float velocityX, float velocityY) {
+        return suppressFlingGesture;
```

Comment:
There's a lot of code in here concerned with the `Slider`. I think it might make sense to try and extract a custom view to handle all this. Maybe a `RangeSlider`?